QuasarHelios2015/src/org/team1540/quasarhelios/Clamp.java

```diff
@@ -0,0 +1,49 @@
+package org.team1540.quasarhelios;
+
+import ccre.channel.BooleanInputPoll;
+import ccre.channel.BooleanOutput;
+import ccre.channel.EventInput;
+import ccre.channel.FloatInputPoll;
+import ccre.channel.FloatOutput;
+import ccre.channel.FloatStatus;
+import ccre.ctrl.BooleanMixing;
+import ccre.ctrl.EventMixing;
+import ccre.ctrl.FloatMixing;
+import ccre.ctrl.PIDControl;
+import ccre.igneous.Igneous;
+
+public class Clamp {
+	public final BooleanOutput openControl;
+	public final FloatOutput heightControl;
+	
+	private FloatStatus min;
+	private FloatStatus max;
+	
+	public Clamp() {
+		
+		BooleanOutput leftClamp = Igneous.makeSolenoid(0);
+		BooleanOutput rightClamp = Igneous.makeSolenoid(1);
+		
+		openControl = BooleanMixing.combine(leftClamp, rightClamp);
+		heightControl = new FloatStatus();
+		min = new FloatStatus(0.0f);
+		max = new FloatStatus(1.0f);
```

Comment:
Consider putting these initializations into the instance variable declarations, for more concise code.